### DISCUSSION TITLE
Publish updated chart w/ configurable nodePort?

### DISCUSSION THREAD

**Discussion by ctoomey** [tchiotludo/akhq#729] [Q&A] [answered]
Hi, saw that https://github.com/tchiotludo/akhq/pull/652 made the nodePort configurable but didn't update the version in `Chart.yaml` and hasn't been published as an updated helm chart. Will you please do that so can use that improvement? Thanks.

**Comment by tchiotludo** [comment, marked as answer]
I hope to have time to release a new version before the end of the month
